feature-terminate-insurance/src/main/kotlin/com/hedvig/android/feature/terminateinsurance/TerminateInsuranceViewModel.kt

```diff
@@ -18,25 +20,37 @@ import kotlinx.datetime.TimeZone
 import kotlinx.datetime.atStartOfDayIn
 import kotlinx.datetime.toInstant
 import kotlinx.datetime.toLocalDateTime
+import java.time.Instant
+import java.time.ZoneId
```

Comment:
Can we try using the kotlinx.datetime.Instant and same for zone? In general they're easier to work with in my experience, and it's nice to be consistent as that is what the Date response from octopus is returning to us anyway.